src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Picture.cs

```diff
@@ -90,7 +90,7 @@ public Size2D GetSize()
         {
             global::System.IntPtr cPtr = Interop.Picture.GetSize(View.getCPtr(this));
```

Comment:
Interop.Picture.GetSize is related to CSharp_Dali_Picture_GetSize:

SWIGEXPORT void *SWIGSTDCALL CSharp_Dali_Picture_GetSize(char *pPicture) {
  Dali::CanvasRenderer::Picture picture;
  Dali::Vector2 result;

  if (!pPicture) {
    SWIG_CSharpSetPendingExceptionArgument(
        SWIG_CSharpArgumentNullException,
        "Attempt to dereference null Dali::CanvasRenderer::Picture", 0);
    return nullptr;
  }
  picture = *(Dali::CanvasRenderer::Picture *)pPicture;

  {
    try {
      result = picture.GetSize();
    }
    CALL_CATCH_EXCEPTION(0);
  }

  return new Dali::Vector2((const Dali::Vector2 &)result);
}